src/components/MenuItem/MenuItem.tsx

```diff
@@ -37,6 +38,12 @@ export const MenuItem: FC<MenuItemProps> = ({
 }) => {
     const isDangerStyle = style === MenuItemStyle.Danger;
 
+    const currentIcon = {
```

Comment:
this might throw an error when `active` is false

Reply:
this will assign `undefined` to the `currentIcon` variable.
react doesn't render `null`, `undefined` or booleans - they are treated the same way